src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/Replacements/DateTimeConverter2.cs

```diff
@@ -32,30 +26,27 @@ public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceT
 
         public override bool CanConvertTo(ITypeDescriptorContext context, Type destinationType)
         {
-            if (destinationType == typeof(string))     
+            if (destinationType == typeof(string))
             {
                 return true;
             }
-            
+
             return base.CanConvertTo(context, destinationType);
         }
 
         public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
-        {
-            return _dateTimeValueSerializer.ConvertFromString( value as string, _valueSerializerContext );
-        }
+            => _dateTimeValueSerializer.ConvertFromString(value as string, null);
```

Comment:
Since `DateTimeValueSerializerContext` doesn't have any functionality, we can just pass null and remove the class here and in the other `DateTimeConverter2`